Sources/XCMetricsClient/Mobius/Domain/MetricsUploaderLoopTypes.swift

```diff
@@ -49,7 +51,8 @@ struct MetricsUploaderModel: Equatable, Hashable {
         isCI: Bool,
         plugins: [XCMetricsPlugin],
         parsedRequests: Set<MetricsUploadRequest> = Set(),
-        awaitingParsingLogResponses: Int = 0
+        awaitingParsingLogResponses: Int = 0,
+        skipNotes: Bool
```

Comment:
nit: does it make sense to add a default value here or better to move `skipNotes` before `parsedRequests`?  We could place args with default values at the end of a list of ars, as Swift recommendeds.
```suggestion
        skipNotes: Bool = false
```

Reply:
Yes, that makes sense!